docs/tutorials/rootless_tutorial.md

```diff
@@ -1,4 +1,4 @@
-![PODMAN logo](../../logo/podman-logo-source.svg)
+g![PODMAN logo](../../logo/podman-logo-source.svg)
```

Comment:
`g` at the start looks like an emacs remnant.

Reply:
Fixed